src/app/spotify-auth/spotify-auth.component.html

```diff
@@ -1,5 +1,16 @@
+          </path>
+        </g>
+      </svg>
```

Comment:
Similar to a previous SVG comment, let's move this into a static file.

Reply:
Done.